.github/workflows/build.yml

```diff
@@ -37,11 +37,11 @@ jobs:
         if: startsWith(matrix.os, 'windows')
 
       - uses: actions/checkout@v3
-      - name: Set up JDK 11
+      - name: Set up JDK 17
```

Comment:
is this a problem? I use jdk 17 per default in all my projects...but i assume i could downgrade to 11 (remove jdk17-textblocks, switch-cases, etc)

Reply:
I'm asking because we still support JDK 11 in the `quarkiverse-parent`, so unless you require JDK 17 features (eg. you have a dependency that requires it), I'd recommend being conservative and keeping it in JDK 11